frontend/src/redux/actions/actions.tsx

```diff
@@ -20,6 +20,15 @@ export const UI_APP_BAR_SET_ACTION = 'UI_APP_BAR_SET_ACTION';
 export const UI_THEME_SET = 'UI_THEME_SET';
 export const UI_RESET_PLUGIN_VIEWS = 'UI_RESET_PLUGIN_VIEWS';
 export const UI_PLUGINS_LOADED = 'UI_PLUGINS_LOADED';
+export const UI_BRANDING_APP_LOGO_COMPONENT = 'UI_BRANDING_APP_LOGO_COMPONENT';
```

Comment:
All the other action names here have verbs, so we should be consistent and call this `UI_BRANDING_SET_APP_LOGO`.